### ISSUE TITLE
Weapons don't consume ammo in survival DS

### ISSUE THREAD

**Issue by ConstyCarlos** [sstixrud/WeaponCore#234]
## Is this a multiplayer or singleplayer bug?
MP DS

## Describe the bug:
after the first reload, the guns no longer consume ammo

## Steps To Reproduce:
use this map for some ready to use samples: https://steamcommunity.com/sharedfiles/filedetails/?id=2267836363
1: get ammo (found inside various containers in/around the turrets
2: load the gun with ammo
3: fire gun...continuously

## Expected behavior:
ammo is consumed, and gun won't fire without it

### User Error Report (if Weapon-Specific) and log files
*WeaponCore lets you generate an error report for a specific weapon, while ingame, hold Ctrl+R and aim the red line at the Weapon that is erroring, and an error report will be generated in WeaponCores log cache: `Users\UserName\AppData\Roaming\SpaceEngineers\Storage\1918681825.sbm_WeaponCore`

Zip up that folder and link it here:

**Comment by pilfit** [comment]
This also happens in SP. the Steps to reproduce it are incomplete.
First load an ammo using fixed gun (only tested with fixed weapons) by hand, fire all rounds then load by hand again. the second time the gun is loaded by hand and ONLY by hand the gun will no longer consume the ammo and fire for free.

**Comment by Qarannia** [comment]
I've found this issue in SP survival and also in MP survival through Steam Friends. On firing the weapon does not consume ammo and on cycling the ammo it actually produces an extra magazine so you end up with more ammo than you originally had. I'm not sure if the two are related but I only saw the extra ammo bug because I was investigating the weapons not reloading properly or consuming ammo in survival.
